cmd/kube-proxy/app/server.go

```diff
@@ -613,6 +613,7 @@ func serveMetrics(bindAddress, proxyMode string, enableProfiling bool, errCh cha
 	})
 
 	//lint:ignore SA1019 See the Metrics Stability Migration KEP
```

Comment:
does this mean all the `lint:ignore SA...` comments are no longer paid attention to?

Reply:
I'm not sure but there is an [issue](https://github.com/golangci/golangci-lint/issues/741) on golangci-lint to that effect. However, I don't see `staticcheck` failures when I run the hack/verify-golangci-lint.sh script on my machine so that seems to suggest that these `//lint:ignore` comments are still being respected or that staticcheck is better than the previous linter and is no longer parking these as errors?

Reply:
Actually, I think I misunderstood what you were saying. You are right that the new staticcheck is not looking at the old `//lint:ignore`, because without the new `//nolint:staticcheck` comment, staticcheck is failing. I will consider updating the PR to remove the now defunct `//lint:ignore` comments and copying their explanations over to the new `//nolint:staticcheck` comments.